Router.svelte

```diff
@@ -533,6 +551,12 @@ const unsubscribeLoc = loc.subscribe(async (newLoc) => {
         // We need to clone the object on every event invocation so we don't risk the object to be modified in the next tick
         dispatchNextTick('routeLoading', Object.assign({}, detail))
 
+        // Set props, if any
+        props = await runInSequence(Object.entries(routesList[i].props).map(([k, v]) => ({
```

Comment:
I think this map is redundant. You should be able to pass `routesList[i].props` directly and check if it's a function in the loop inside `runInSequence`. This way, we avoid re-allocating a new array.